examples/tfengine/generated/team/live/example-prod-networks/resources/main.tf

```diff
@@ -58,18 +58,18 @@ module "example_network" {
   project_id   = var.project_id
   subnets = [
     {
-      subnet_name            = "example-bastion-subnet"
-      subnet_ip              = "10.1.0.0/16"
-      subnet_region          = "us-central1"
-      subnet_flow_logs       = true
-      subnets_private_access = true
+      subnet_name           = "example-bastion-subnet"
+      subnet_ip             = "10.1.0.0/16"
+      subnet_region         = "us-central1"
+      subnet_flow_logs      = true
+      subnet_private_access = true
```

Comment:
Do you know why this missed our test? Shouldn't terraform plan fail?

Reply:
Nevermind, subnets are implemented as a map:https://github.com/terraform-google-modules/terraform-google-network/blob/master/variables.tf#L38

Reply:
Yea, I think once the named object is there, we can make it more restrict as well in the CFT module.